HTTPServer.driver/Logger.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using HTTPServer.Core;
+using System.Text;
+
+namespace HTTPServer.app
+{
+    public class Logger
+    {
+        public void LogData(string loggingMessage)
+        {
+            Console.Write(loggingMessage);
+        }
+
+        public string LogMessage(Request request, Reply reply)
+        {
+            var responseCode = GetResponseCode(reply.StartingLine);
+            var logMessage = DateTime.Now.ToString() + " " + request.Method + " " + request.Uri + " "
+                + request.HttpVersion + " " + responseCode + "\r\n";
+            return logMessage;
+        }
+
+        private string GetResponseCode(byte[] response)
+        {
+            var firstLine = Encoding.UTF8.GetString(response);
+            var requestLine = firstLine.Split(' ', ' ');
```

Comment:
It's not.  I changed this code quite a bit from what it originally was and didn't change the variable names.  It is getting the response code from the first line of a reply message.  It is one of the things that is needed in the logging message.